website/docs/r/iam_server_certificate.html.markdown

```diff
@@ -116,6 +116,13 @@ The following arguments are supported:
 * `name` - The name of the Server Certificate
 * `arn` - The Amazon Resource Name (ARN) specifying the server certificate.
 
+## Import
+
+IAM Server Certificates can be imported using the `name`, e.g.
+
+```
+$ terraform import aws_iam_server_certificate.sertificate example.com-certificate-until-2018
```

Comment:
@Ninir I am going to make the change on this branch and then merge this - I believe it was a typo :)